docs/advanced/images-adapters/twicpics.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 1
+title: TwicPics
+description:
+  Deliver perfectly sized images in real-time to create visual-rich experiences.
+tags:
+  - Image Adapter
+  - TwicPics
+  - Since version 2.20.x
+---
+
+import ContactLink from "@site/src/components/ContactLink";
+
```

Comment:
similar to Cloudinary's: there is a missing `<SinceVersion>` component here.

If we want to use tags for it (could be a good idea), I think it has to be done separately and globally.
Possibly, we could automate the `<SinceVersion>` at layout level if we detect the tag of a version.

That way, we could have no duplication and consistency.

But this is out of scope of this PR, hence the suggestion of adding the component manually like everywhere else, for consistency.

Reply:
Yea so this was prior to the `SinceVersion` component which led to the creation of that component.

I still need to merge it to the next branch and rebase, I will handle that next, just finishing up the demo's